clients/views.py

```diff
@@ -26,28 +26,22 @@
 
 
 
-class ClientsView(LoginRequiredMixin,TemplateView):
+class ClientListView(LoginRequiredMixin,TemplateView):
 	template_name = 'clients/all_client.html'
 
 	def get_context_data(self,request,**kwargs):
 		#import pdb; pdb.set_trace()
 		context = super().get_context_data(**kwargs)
-		try :
-			clients = Client.objects.filter( owner=request.user)
-		except:
-			clients = Client.objects.all()
-		try:
-			context['clients'] = Client.objects.filter(owner=request.user)
-		except:
-			context['clients'] = Client.objects.all()
+		clients = Client.objects.filter( owner=request.user)
+
+		context['clients'] = Client.objects.filter(owner=request.user)
+
 		#context['invoices'] = Invoice.objects.filter(user_createdby=request.user.company_name, client_name=Client.objects.all())
 		#context['client_fil'] = 
 		query = request.GET.get("q")
 		if query:
-			try:
-				clients = clients.filter(Q(display_name__icontains=query,  owner=request.user) )
-			except:
-				clients = clients.filter(Q(display_name__icontains=query) )
+			clients = clients.filter(Q(display_name__icontains=query,  owner=request.user) )
```

Comment:
what is the use of `Q`. read and understand https://docs.djangoproject.com/en/1.7/ref/models/queries/#q-objects